### ISSUE TITLE
[Question] Version docs translation

### ISSUE THREAD

**Issue by Mister-Hope** [pnpm/pnpm.io#599]
Hi @zkochan and the pnpm maintainers,

I’m reaching out because I’m quite confused about how translations are being handled for versioned documentation in Crowdin.

In November, I suggested that all translations for the content inside the `docs` folder, and versioned documentation `versioned_docs/version_9x` should be automatically generated from the docs folder during each new publish. Based on this, I expected the translations to sync automatically, but unfortunately, this doesn’t seem to be the case.

![image](https://github.com/user-attachments/assets/1cc6c833-7a4c-4a15-99c2-3f4bb090bb7a)

As shown in the screenshot above, after declaring the `docs` folder as `version_10x`, the translations have not been synced properly. The translation progress for `version_9x` is still stuck in its previous state, and some translations, especially those updated in the English content, appear to have been lost. For example, code block changes in the original `docker.md` file have resulted in existing translation suggestions (which I submitted) disappearing. This behavior is unexpected, and I would appreciate it if you could look into this issue, as I was not expecting to manually re-translate everything for `version_9x`.

Additionally, I’m frustrated that many translations for the newly generated `version_10x` documentation have still not been synced. As of yesterday, the approved (suggest) translation rate for the Chinese docs folder was over 75%/85%, yet only about 35%/68% of translations have been synced for `version_10x`. Why is this the case, and why hasn’t the translation progress for `version_10x` been reflected as expected?

I hope someone from the team can review this issue and help clarify why these syncing problems are happening. Your assistance would be greatly appreciated.

Thank you for your attention to this matter!

Best regards,  Mr.Hope

**Comment by Mister-Hope** [comment]
![image](https://github.com/user-attachments/assets/2f724e07-e5cb-4df3-95c2-8d722fd4dcb7)
![image](https://github.com/user-attachments/assets/abda9488-0666-4da3-a35d-1bba5dcfe575)

Since I’m performing a detailed line-by-line check of the docs, I’ve noticed that some newly created files aren’t properly setting hidden strings at all, and there are missing ones in existing files. It would be great if I had the ability to help address this more effectively.

So I would like to grant permissions to mark strings as hidden, like in the screenshots below. This would help prevent newly created files from missing hidden strings and ensure that existing files are fully up to date.

**Comment by Mister-Hope** [comment]
I would also want to add some word in the glossary, to help translation unique and easier, e.g.:

- common technical words like: fork, directory, semver, registry, patch, catalog, peerDependencies
- common headings like Commands, Options, etc.

**Comment by zkochan** [comment]
Right, this problem have existed for years. Every time we release a new version all the previous translations are not applied automatically. TBH, I am not happy with crowdin, it has many issues. I regret that we have localization at all as it involves a lot of manual work.

**Comment by Mister-Hope** [comment]
Thank you for your prompt response. I have a couple of follow-up questions, and I would greatly appreciate your clarification on these points:

1. Is it possible to grant the ability to mark strings as "hidden" or "need translation," as well as manage glossary items?
2. Can translations be synced by exporting them from the docs folder, then re-importing them into the versioned_docs/version_10x folder and approving everything using the "select all" option? If this process is feasible, I will notify you once I have finished reviewing, completing, and improving the contents of the docs folder.
